src/components/forms/CharacterCount/CharacterCount.stories.tsx

```diff
@@ -0,0 +1,95 @@
+import React from 'react'
+import { CharacterCount } from './CharacterCount'
+import { Form } from '../Form/Form'
+import { FormGroup } from '../FormGroup/FormGroup'
+import { Label } from '../Label/Label'
+
+export default {
+  title: 'Forms/CharacterCount',
+  parameters: {
+    info: `
+USWDS 2.0 Character count component
+
+Source: https://designsystem.digital.gov/components/form-controls/#character-count
+`,
+  },
+}
+
+export const textInput = (): React.ReactElement => (
+  <FormGroup>
```

Comment:
I think we want form component stories to be inside a `Form`, not just a `FormGroup`